gulpfile.js

```diff
@@ -102,7 +114,7 @@ gulp.task('build-bundle-dev', ['devpack'], bundle.bind(null, true));
 gulp.task('build-bundle-prod', ['webpack'], bundle.bind(null, false));
 gulp.task('bundle', bundle.bind(null, false)); // used for just concatenating pre-built files with no build step
 
-gulp.task('devpack', ['clean'], function () {
+gulp.task('devpack', ['clean-dev'], function () {
```

Comment:
I think this introduces a race condition into `gulp serve`, doesn't it?

`serve` depends on `build-bundle-dev` and `test`. `test` depends on `clean`, but `build-bundle-dev` depends on `clean-dev`.

Since both depend on `clean` currently, gulp is smart enough to manage the dependency graph and make sure `clean` finishes before `build-bundle-dev` and `test` start.

By changing it to `clean-dev`, gulp won't know... so it's possible that the `clean` from `test` wipes out the code _after_ `build-bundle-dev`has already finished, leaving you with no code to `serve`.